scenarios/17_Sneaking_out_of_Raedwood.cfg

```diff
@@ -368,10 +372,178 @@
         {MSG_Quivre (_"There is a gossip that this unknown curse put on our women will end. " + _"Everyone who has forsaken our old ways and is too underling-friendly will soon be as dead as our old king. Or as that mixling, ten-days king, Gawen.")}
         {MESSAGE (Bar O Raednon) (portraits/akladian_lord.png) (Bar O Raednon) _"Cousin, do not utter that name! I've heard he was seen walking the face of the earth... as if... he had come back from the grave... or perhaps undead..."}
         {MSG_Ruvio _"We have to get past these guards. Akladians set up a line of guards around the forest some time ago, when our rebellion was more visible and having some success."}
+        # FIXME: what if Gawen didn't recruit/recall the max amount possible in the previous scenario?
+#ifdef EASY
+        #po: EASY difficulty:
+        {MSGm_Gawen _"I want to avoid fighting. I only have eighteen seasoned troops, and in this neigborhood I can recruit only peasants. I won't be able to find replacements for any of my veterans, and this is just the beginning of our quest."}
+#endif
+#ifdef NORMAL
+        #po: NORMAL difficulty:
+        {MSGm_Gawen _"I want to avoid fighting. I only have fifteen seasoned troops, and in this neigborhood I can recruit only peasants. I won't be able to find replacements for any of my veterans, and this is just the beginning of our quest."}
+#endif
+#ifdef HARD
+        #po: HARD difficulty:
         {MSGm_Gawen _"I want to avoid fighting. I only have twelve seasoned troops, and in this neigborhood I can recruit only peasants. I won't be able to find replacements for any of my veterans, and this is just the beginning of our quest."}
-        {MSG_Ruvio _"If so, then it is up to you to figure out a strategy which will avoid any losses. While will have to pass near the guards, and may skirmish with them, there is no need to get into a stand-up fight with them."}
+#endif
+#ifdef NIGHTMARE
+        #po: unused:
+        {MSGm_Gawen _"I want to avoid fighting. I barely have any seasoned troops, and in this neigborhood I can recruit only peasants. I won't be able to find replacements for any of my veterans, and this is just the beginning of our quest."}
+#endif
+        {MSG_Ruvio _"If so, then it is up to you to figure out a strategy which will avoid any losses. While we will have to pass near the guards, and may skirmish with them, there is no need to get into a stand-up fight with them."}
```

Comment:
What is a stand-up fight? You mean a direct fight?

Reply:
"stand-up" fight was in the original string, too (see the left side of this diff), so you'd have to ask szopen...

Reply:
You can leave it for now then.